cvsd.gd

```diff
@@ -0,0 +1,16 @@
+extends Node
```

Comment:
is this cvsd.gd file suppose to be here? Is it being used?

Reply:
Oh sorry, I don't know how that got there, fixed